### PR TITLE
[Automation] Add tc to validate cephadm adopt fails to start OSDs whe…

### PR DESCRIPTION
**Pull request by pranavprakash20**
…n dmcrypt: true

Adding tc to validate the ceph cluster when created using a custom name
and dmcrypt is set a True

Bz: https://bugzilla.redhat.com/show_bug.cgi?id=2058038
Test run result : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-AIGU4Q/

Signed-off-by: Pranav <prprakas@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container_custom_name.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+               fetch_directory: ~/fetch
+               copy_admin_key: true
+               containerized_deployment: true
+               upgrade_ceph_packages: True
+               dashboard_enabled: True
+               dashboard_admin_user: admin
+               dashboard_admin_password: p@ssw0rd
+               grafana_admin_user: admin
+               grafana_admin_password: p@ssw0rd
+               node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
```
Comment (sunilangadi2) [inline suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container_custom_name.yaml:107]: In 5.2 onwards monitoring stack images got updated to `v4.10` refer: https://bugzilla.redhat.com/show_bug.cgi?id=2065443#c6
hence can you please use the same images here?

**Review by sunilangadi2 — CHANGES_REQUESTED** [review]

File: suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container_custom_name.yaml (hunk 1 above)
Comment (pranavprakash20) [reply suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container_custom_name.yaml:107]: Ack. Noted, the same shall be updated

**Comment by vamahaja** [comment]
Changes looks good to me. Please update monitoring stack image tag as suggested by sunil.

**Review by vamahaja — CHANGES_REQUESTED** [review]
@manasagowri @sunilangadi2 please review

**Review by sunilangadi2 — APPROVED** [review]
LGTM.

**Pull request merged**